### PR TITLE
Update MediaServices to package-metadata-2022-08

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2022-09-14)
 
 ### Features Added
 
+- Upgrade to `package-metadata-2022-08`
```
Comment (Yao725) [inline sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md:7]: Is it better to use the real API version here?
```suggestion
- Upgraded the API version to `2022-08-01`
```

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+
 ### Breaking Changes
 
+- Renamed `TransformOutputsPriority` to `MediaTransformOutputsPriority`.
+- Renamed `ContentKeyPolicyPreference` to `ContentKeyPolicyOption:`.
```
Comment (Yao725) [inline sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md:12]: Please remove the extra colon at the end of this line.

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+- Renamed `GetContainerSasContent` to `GetStorageContainersContent:`.
```
Comment (Yao725) [inline sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md:13]: Same as above

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
+- Renamed `MediaPreset` to `MediaTransformPreset:`.
```
Comment (Yao725) [inline sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md:14]: Same as above.

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md [hunk 5]
```diff
@@ -1,11 +1,19 @@
...
+- Remove all get LRO operation status / result methods which is nativly supported by the LRO object ArmOperation.
```
Comment (Yao725) [inline sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md:15]:
```suggestion
- Removed all get LRO operation status / result methods which is nativly supported by the LRO object ArmOperation.
```

**Review by Yao725 — APPROVED** [review]
Code change looks good to me.

**Review by fengzhou-msft — APPROVED** [review]

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md]: fixed

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md (hunk 3 above)
Comment (ArthurMa1978) [reply sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md]: fixed

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md]: fixed

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md (hunk 4 above)
Comment (ArthurMa1978) [reply sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md]: fixed

File: sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md (hunk 5 above)
Comment (ArthurMa1978) [reply sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md]: fixed

**Pull request merged**